index.js

```diff
@@ -122,18 +122,30 @@ exports.getCommitLanguages = (visibility, token) => {
 
                 // For each file in the commit files
                 _.each(commits, (commit) => {
+                    var commitLangs = []; // Store all the languages present in commit
                     _.each(commit.files, (file) => {
                         var language = detect.filename(file.filename);
                         if (language) {
+                            // Create empty object to hold total values
+                            if (!totals[language]) totals[language] = {
+                                bytes: 0,
+                                commits: 0
+                            };
+
+                            // Add one to the language commit count if we haven't already
+                            if (commitLangs.indexOf(language) == -1) {
```

Comment:
Did you consider/shoot down `!commitLangs.includes(language)`? Purely curious...although it is an es6.